docs/UserGuide.adoc

```diff
@@ -11,226 +12,341 @@ ifdef::env-github[]
-* Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+* Words in `UPPER_CASE` are the parameters to be supplied by the user. +
+e.g. In `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+* Items in square brackets are optional. +
+e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+* Items with `…`​ after them can be used multiple times, or none at all. +
+e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Parameters can be in any order. +
+e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+* For your convenience, many commands have an alias. +
+eg. `list` and `l` will both show a list of all employees in PTMan.
 ====
 
-=== Viewing help : `help`
+=== Shared Features
+These commands can be executed by both employees and employers.
 
-Format: `help`
+==== Viewing help: `help`
 
-=== Adding a person: `add`
+Displays the help window. +
+*Format:* `help`
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+==== Locating an employee by name: `find`
+
+Finds employees whose names contain any of the given keywords. +
+*Format:* `find KEYWORD [MORE_KEYWORDS]` +
+*Shorthand:* `f KEYWORD [MORE_KEYWORDS]`
+
+****
+* The search is case insensitive. e.g `hans` will match `Hans`
+* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+* Only the name is searched.
+* Only full words will be matched e.g. `Han` will not match `Hans`
+* Employees matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+****
+
+Examples:
+
+* `find John` +
+Returns `john` and `John Doe`.
+* `find Betsy Tim John` +
+Returns any employee having names `Betsy`, `Tim`, or `John`.
+
+==== Listing all employees : `list`
+
+Shows a list of all employees in PTMan. +
+*Format:* `list` +
+*Shorthand:* `l`
+
+==== Listing entered commands: `history`
+
+Lists all the commands that you have entered in reverse chronological order. +
+*Format:* `history` +
+*Shorthand:* `h`
+
+[NOTE]
+====
+Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
+====
+
+==== Exiting the program: `exit`
+
+Exits the app. +
+*Format:* `exit`
+
+=== Employer Features
+These commands are privileged, and can only be executed by the employer. These commands requires the employer's admin password.
+
+[CAUTION]
+Admin password must always be entered as the *last* parameter of a command.
+
+==== Adding an employee: `add`
+
+Adds an employee to PTMan. +
+*Format:* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS s/SALARY [t/TAG]... pw/ADMIN_PASSWORD` +
```

Comment:
1. Do you mean that for every employee to be added, the employer has to enter his password? If so, think how you can make employers' life simple

2. Are you asking for the password in plain text? :confused: